File: samples/sandbox/app/src/main/java/com/scichart/docsandbox/examples/java/axisAPIs/AxisAlignmentSettingAxisAlignment.java
package com.scichart.docsandbox.examples.java.axisAPIs;

import androidx.annotation.NonNull;

import com.scichart.charting.visuals.SciChartSurface;
import com.scichart.charting.visuals.axes.AxisAlignment;
import com.scichart.charting.visuals.axes.NumericAxis;
import com.scichart.docsandbox.core.ExampleDefinition;
import com.scichart.docsandbox.examples.base.SingleChart2DFragment;

@ExampleDefinition()
public class AxisAlignmentSettingAxisAlignment extends SingleChart2DFragment {
    @Override
    protected void initExample(@NonNull SciChartSurface surface) { }

    void addAxisAlignment() {
        // <AddAxisAlignment>
        final NumericAxis xTopAxis = new NumericAxis(getContext());
        xTopAxis.setAxisId("TopAxisId");
        xTopAxis.setAxisAlignment(AxisAlignment.Top);

        final NumericAxis xBottomAxis = new NumericAxis(getContext());
        xBottomAxis.setAxisId("BottomAxisId");
        xBottomAxis.setAxisAlignment(AxisAlignment.Bottom);

        final NumericAxis yLeftAxis = new NumericAxis(getContext());
        yLeftAxis.setAxisId("LeftAxisId");
        yLeftAxis.setAxisAlignment(AxisAlignment.Left);

        final NumericAxis yRightAxis = new NumericAxis(getContext());
        yRightAxis.setAxisId("RightAxisId");
        yRightAxis.setAxisAlignment(AxisAlignment.Right);
        // </AddAxisAlignment>
    }
}

File: samples/sandbox/app/src/main/java/com/scichart/docsandbox/examples/kotlin/axisAPIs/AxisAlignmentSettingAxisAlignment.kt
package com.scichart.docsandbox.examples.kotlin.axisAPIs

import com.scichart.charting.visuals.SciChartSurface
import com.scichart.charting.visuals.axes.AxisAlignment
import com.scichart.charting.visuals.axes.NumericAxis
import com.scichart.docsandbox.core.ExampleDefinition
import com.scichart.docsandbox.examples.base.SingleChart2DFragment

@ExampleDefinition()
class AxisAlignmentSettingAxisAlignment : SingleChart2DFragment() {
    override fun initExample(surface: SciChartSurface) {}

    fun addAxisAlignment() {
        // <AddAxisAlignment>
        val xTopAxis = NumericAxis(context)
        xTopAxis.axisId = "TopAxisId"
        xTopAxis.axisAlignment = AxisAlignment.Top

        val xBottomAxis = NumericAxis(context)
        xBottomAxis.axisId = "BottomAxisId"
        xBottomAxis.axisAlignment = AxisAlignment.Bottom

        val yLeftAxis = NumericAxis(context)
        yLeftAxis.axisId = "LeftAxisId"
        yLeftAxis.axisAlignment = AxisAlignment.Left

        val yRightAxis = NumericAxis(context)
        yRightAxis.axisId = "RightAxisId"
        yRightAxis.axisAlignment = AxisAlignment.Right
        // </AddAxisAlignment>
    }
}
